### PR TITLE
📝 Improve dev readme

### PR DESCRIPTION
**Pull request by wvangeit**
## What do these changes do?

Add instruction on how to perform dev build.
Clean up the READMEs of api-server and director-v2 services

## Related issue/s

N/A

## How to test

Read the READMEs ;-)

## DevOps Checklist

N/A

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
Cool. Thx so much for this update!

**Review by sanderegg — APPROVED** [review]

File: services/api-server/README.md [hunk 1]
```diff
@@ -1,40 +1,15 @@
...
-cd services/api-server
-make install-dev
-```
-
-Then
-
-```cmd
-make run-devel
-```
+Since services are often heavily interconnected, it's best to build and run the entire osparc repo in the development mode.
```
Comment (bisgaard-itis) [inline services/api-server/README.md:7]: I would rephrase: "Since the API server communicates with (almost) all other services, running it requires running the entire osparc (see the section [development build](../../README.md#development-build) section of the main README)"

**Review by bisgaard-itis — APPROVED** [review]
Thanks for improving our doc! Just a single suggestion from my side

File: services/api-server/README.md (hunk 1 above)
Comment (wvangeit) [reply services/api-server/README.md]: Thx, done.

**Review by GitHK — APPROVED** [review]
